python/Upload_CV_List.py

```diff
@@ -0,0 +1,42 @@
+# Preliminaries
+import pandas as pd
+import argparse
+import synapseclient
+from synapseclient import Project, File, Folder
+from synapseclient import Schema, Column, Table, build_table, Row, RowSet, as_table_columns
+from pathlib import Path
+
+# Login to  Synapse
+syn = synapseclient.Synapse()
+syn.login()
+
+# Get Arguments - Synapse project id and file path
+parser = argparse.ArgumentParser(description = 'Get synapse project id, file path, and name of table')
+parser.add_argument('project_id',type=str,help='Synapse Project ID')
```

Comment:
There should be a space after each comma (except for trailing commas).

```suggestion
parser.add_argument('project_id', type=str, help='Synapse Project ID')
```